Dear/Dear/Controllers/HospitalViewController.swift

```diff
@@ -18,25 +19,77 @@ class HospitalViewController: UIViewController {
         pickerView.dataSource = self
         
         hospitalTextField.inputView = pickerView
+        hospitalTextField.borderStyle = .none
+        hospitalTextField.layer.cornerRadius = 10
+        hospitalTextField.backgroundColor = UIColor(named: "TextFieldBackgroundColor")
         hospitalTextField.textAlignment = .left
+        hospitalTextField.addLeftPadding()
+        hospitalTextField.text = hospitals[0].hospitalName
+        
+        let backButton = UIBarButtonItem(title: "이전", style: .plain, target: self, action: #selector(popToPrevious))
+        backButton.tintColor = .black
+        navigationItem.setLeftBarButton(backButton, animated: true)
+        
+        let confirmButton = UIBarButtonItem(title: "완료", style: .plain, target: self, action: #selector(pushToNext))
+        confirmButton.tintColor = .black
+        navigationItem.setRightBarButton(confirmButton, animated: true)
+        
+        setup()
+    }
+    
+    // Navigation 상에서 뒤로 가기 위한 액션 구현
+    @objc private func popToPrevious() {
+        navigationController?.popViewController(animated: true)
+    }
+    
+    @objc private func pushToNext() {
+        let user = UserDefaults.standard.string(forKey: "user")
+        
+        if hospitalTextField.text != "병원을 선택해주세요." {
```

Comment:
```suggestion
        if hospitalTextField.text == "병원을 선택해주세요." { return }
```

아래의 코드들은 인덴트 없이 읽기 쉬워질 것 같아요